PrebidMobile/AdUnits/MultiformatAdUnit/PrebidRequest.swift

```diff
@@ -64,6 +64,20 @@ public class PrebidRequest: NSObject {
         self.gpid = gpid
     }
     
+    // MARK: Arbitrary ORTB Configuration
+    
+    /// Sets the impression-level OpenRTB configuration string for the ad unit.
+    ///
+    /// - Parameter ortbObject: The  impression-level OpenRTB configuration string to set. Can be `nil` to clear the configuration.
+    public func setImpressionORTBConfig(_ ortbConfig: String?) {
```

Comment:
Please add an example of how this method is used in the demo apps. 

Let's set: 
- `imp.bidfloor=10` 
- `imp.banner.battr` = [1,2,3,4]